index.js

```diff
@@ -22,12 +24,17 @@ const setup = (options = {}) => {
         ...options
     });
 
+    if (options.hostCallBackMiddlewareSetup && typeof options.hostCallBackMiddlewareSetup === 'function') {
+        options.hostCallBackMiddlewareSetup(app);
+    }
+
     const staticRouter = express.Router();
     app.use(staticRouter);
 
     if (options.session !== false) middleware.session(app, {
         ...config.get('session'),
-        ...options.session
+        ...options.session,
+        trustProxy: app.get('trust proxy')
```

Comment:
This shouldn't need to be specified, as when proxy is undefined session will use the app's trust proxy anyway.

Reply:
Thanks, didn't realise once x-forwarded-proto was set to https that req.secure is evaulated as true!